### ISSUE TITLE
Deprecation notice on Chrome Store

### ISSUE THREAD

**Issue by Shion1305** [mxrch/ghunt_companion#5]
Chrome says GHunt Companion may soon no longer be supported.
<img width="811" alt="Screenshot 2024-08-02 at 11 02 22" src="https://github.com/user-attachments/assets/a81be457-cc63-48c3-b103-a56d8012c192">
<img width="1113" alt="Screenshot 2024-08-02 at 11 03 41" src="https://github.com/user-attachments/assets/9356e294-35bc-4dbd-9983-d1a9120367f6">

**Comment by DrAnonym666** [comment]
yes because policy change....many Addons are not more available at CHROME

**Comment by KenB773** [comment]
Is there an alternative or workaround?

**Comment by Shion1305** [comment]
Chrome Extensions Ver 2 is no longer available.
<img width="1161" alt="Image" src="https://github.com/user-attachments/assets/bf39a738-83e5-4098-ae16-77637f372bc4" />

Let's identify what our challenges will be on migrating this extension to ver 3

**Comment by toby-bro** [comment]
checked on chromium and same issue

**Comment by rootkea** [comment]
@mxrch, will it be possible to have this resolved for Chrome/Chromium? Thanks!

**Comment by rootkea** [comment]
I just noticed. There seems to be an existing PR already addressing this issue: #7
